packages/common/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
+    "outDir": "dist",
+    "baseUrl": "./src",
+    "module": "esnext",
+    "target": "es5",
+    "lib": ["es6", "dom", "es2016", "es2017"],
+    "sourceMap": true,
+    "allowJs": false,
+    "jsx": "react-jsx",
```

Comment:
Oh yeah, don't need jsx here either

Reply:
nice also, thinking we merge our tsconfigs somewhere to, like eslint and maybe prettier?